domains/asgar.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio",
+    "repo": "https://github.com/aalishali/portfolio",
+    "owner": {
+        "username": "aalishali",
+        "email": "foreverasgar@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://aalishali.github.io/portfolio/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "aalishali.github.io"
```